src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -77,14 +78,14 @@ Instance configuration is generated by Che and is updated by our internal config
 ----
 
 [id="jdbc-configuration"]
-== JDBC Configuration
+== JDBC configuration
 
-Eclipse Che uses http://www.h2database.com/html/main.html[H2] for single-user builds and PostgreSQL database in a multi-user flavor.
+Eclipse Che uses http://www.h2database.com/html/main.html[H2] for single-user builds and PostgreSQL database in the multi-user flavor.
```

Comment:
Is "multi-user" referred to as a "flavor" often?

Reply:
Nope. Just this one instance.